axelor-business-project/src/main/java/com/axelor/apps/businessproject/service/TimesheetProjectServiceImpl.java

```diff
@@ -179,4 +181,15 @@ public BigDecimal computeDurationForCustomer(TimesheetLine timesheetLine) throws
     return timesheetLineService.computeHoursDuration(
         timesheetLine.getTimesheet(), timesheetLine.getDurationForCustomer(), true);
   }
+
+  @Override
+  protected TimesheetLine createTimeSheetLineFromPPT(
+      Timesheet timesheet, ProjectPlanningTime projectPlanningTime) throws AxelorException {
+    TimesheetLine line = super.createTimeSheetLineFromPPT(timesheet, projectPlanningTime);
+    if (projectPlanningTime.getProjectTask().getInvoicingType()
```

Comment:
project task is nullable on projectPlanningTime, please add a check to avoid NPE